src/pages/Home.js

```diff
@@ -0,0 +1,20 @@
+import { checkForToken, addTokenToLocalStorage } from '../lib/localStorage';
+import { useHistory } from 'react-router-dom';
+
+export default function Home() {
+  const isThereToken = checkForToken('token', '');
+  const history = useHistory();
+
+  function handleClick() {
+    addTokenToLocalStorage('token');
+    history.push('/list');
+  }
+
+  return (
+    <main>
+      {isThereToken ? history.push('/list') : ''}
+      <h1>Welcome screen</h1>
+      <button onClick={handleClick}>Add List</button>
+    </main>
+  );
+}
```

Comment:
Y'all knocked this out of the park! Nice job!

Reply:
YAY! :D